src/utilities/theme/utils.ts

```diff
@@ -258,6 +259,29 @@ function successColors(color: HSLAColor, lightSurface: boolean) {
   };
 }
 
+function opacityColors() {
+  return {
+    opacityShallow: {
```

Comment:
```suggestion
    diffuseShadow: {
```

Reply:
Though I found myself explaining this as:

```suggestion
    shadowFromAmbientLight: {
```